storage/rocksdb/rdb_datadic.h

```diff
@@ -742,6 +742,10 @@ class Rdb_ddl_manager {
   my_core::HASH m_ddl_hash; // Contains Rdb_tbl_def elements
   // maps index id to <table_name, index number>
   std::map<GL_INDEX_ID, std::pair<std::string, uint>> m_index_num_to_keydef;
+
+  //maps index id to key definitons not yet committed to data dictionary
```

Comment:
nit typo: definitions and space between // and map.

Could also mentioned this is mainly used by alter table.